src/cloud-element-templates/cmd/ChangeElementTemplateHandler.js

```diff
@@ -848,6 +928,16 @@ export default class ChangeElementTemplateHandler {
     return bo && bo.get('messageRef');
   }
 
+  _getSignal(element) {
+    let bo = getBusinessObject(element);
+
+    if (is(bo, 'bpmn:Event')) {
+      bo = bo.get('eventDefinitions')[0];
+    }
+
+    return bo && bo.get('signalRef');
+  }
```

Comment:
This is misleading because only `bpmn:Event`s can have a signalRef (unlike messages which can referenced in some tasks).

```suggestion
  _getSignal(element) {
    let bo = getBusinessObject(element);

    if (!is(bo, 'bpmn:Event')) {
      return;
    }
    
    const eventDefinition = bo.get('eventDefinitions')[0];

    return eventDefinition && eventDefinition.get('signalRef');
  }
```